### DISCUSSION TITLE
The future of jellyseerr?

### DISCUSSION THREAD

**Discussion by CEbbinghaus** [seerr-team/seerr#394] [General]
TLDR: Overseer are removing their plex limitation in sct/overseerr#3015 which could make Jellyseer obsolete.

So I have been looking all around for the solution to one of my problems which are sorting different TV shows into their respective categories on disk (anime, cartoon, TV, etc..). and came across sct/overseerr#351 which addresses this exact problem. Turns out the feature got written and released in [v1.17.0](https://github.com/sct/overseerr/releases/tag/v1.17.0) which prompted me to do some digging on figuring out if there was some way to get those changes merged into Jellyseer (It already is. Thanks to Fallenbagel for correcting me). It didn't take me long to stumble upon sct/overseerr#3015 which unlike it first appeared wasn't dead but instead currently in progress. For anyone too lazy to click on the link it removes the tight coupling between Overseerr & Plex and "Opens the floodgates" to other media servers (Jellyfin, Emby, etc...).

This seems to be quite a "threat" to jellyseer as it removes the main draw for using it over overseerr, especially with the number of features being added. As such I was wondering what 
* A. the future of this project will become in a post Jellyfin + Overseer world.
* B. whether the maintainers/contributors for this project would invest the time to bring Jellyfin to overseer themselves (since that looks to be left to others)?
* C. if the two projects would/could unite and merge back into one where the Jellyseerr & Overseerr teams both maintain the single codebase.

I am VERY fresh to anything even remotely close to Servarr and as such don't have a deep understanding of any grudges but from sct/overseerr#288 it seems clear that "Plexseer" is clearly not an endearing term.

**Comment by fallenbagel** [comment]
We are working with the overseerr on the plex decouple and bringing jellyfin/emby support into it. As for the merging thing, we routinely merge in all features implemented in overseerr so anything on overseerr is already in jellyseerr.

**Reply by CEbbinghaus** [reply]
Love to hear although in this case the feature of getting to choose the directory doesn't yet exist in jellyseer. Glad that the jellyseer team are working on the decoupling (and presumably jellyfin support). As such will jellyseer be sunset once overseer has full jellyfin support?

**Reply by fallenbagel** [reply]
> Love to hear although in tbis case the feature of getting to choose the directory doesn't yet exist in jellyseer. 

What do you mean? Can you show a screenshot? Because we are in feature parity with overseerr and we do have choose directory feature in jellyseerr same as overseerr. 

Edit: I just went over the issue and the release notes you linked. I think you are fundamentally wrong idea on how the feature works. What you are referring to is the `Advanced Requests` feature. This does exist in jellyseerr. You have to make sure that your users have the permission for it.

![Screenshot_20230531_051753_Chrome Dev](https://github.com/Fallenbagel/jellyseerr/assets/98979876/49f7ffec-6fe9-43f5-8024-cd5915917e3c)

(This is a screenshot of a user who has the Advanced request permission. As you can see you can select the folder and quality BEFORW requesting.)

**Reply by CEbbinghaus** [reply]
So it seems that it indeed has the root folders. That seems to have just been some conifg missing on my end (I appreciate the help). The main point of that post was more on what the Jellyseer project becomes once it looses its main draw anyways 😊

**Reply by fallenbagel** [reply]
No worries c:
As mentioned earlier, we are working closely with Overseerr team to bring support for jellyfin/emby (scenario C).
Overseerr never was, nor never will be a "threat" to jellyseerr as jellyseerr would not be possible without it🫡

**Reply by CEbbinghaus** [reply]
threat might have been the wrong word to use. But the fork exists because it didn't have jellyfin support right? You said you are working to bring Jellyfin support to Overseerr & will be working together (scenario C) but that to me implies that after such work is done the Jellyseerr fork ceases to exist and gets archived and development focuses back on the main branch. Is that correct?

**Reply by Continous** [reply]
It sounds more like the projects will be developed concurrently, sharing any features as necessary/able.

**Reply by CEbbinghaus** [reply]
right. So the post Overseer + Jellyfin world will have Overseerr with Jellyfin support & Jellyseerr for jellyfin only.

**Reply by fallenbagel** [reply]
> right. So the post Overseer + Jellyfin world will have Overseerr with Jellyfin support & Jellyseerr for jellyfin only. 

No.

**Reply by DuvelCorp** [reply]
It has taken 3 years to the overseer team to finally do a beginning of something for non-Plex users, and even though they speak about decoupling, I still not see a single mention of "Jellyfin" in their plans. I have the feeling that it will take them another 3 years to finally have something decent, ie 100% compatible with Jellyfin like Jellyseer is.

And I wouldn't move my ass from here in any case !!!

**Reply by CEbbinghaus** [reply]
I just saw `seerr` mentioned and got a bit confused, looked it up and it looks like this problem has solved itself, Jellyseerr won out in the end :3

**Comment by CEbbinghaus** [comment]
This has been solved now with the merging of overseerr and jellyseerr development. Love the work and keep it up <3
